virtwho.py

```diff
@@ -175,50 +240,67 @@ def run(self):
             return
 
         result = {}
+        report = None
         while not self.terminate_event.is_set():
             # Wait for incoming report from virt backend
             try:
-                report = self.queue.get(block=True, timeout=None)
+                report = self.queue.get_nowait()
```

Comment:
Having timeouts for each Job allows us to check and record the status of async jobs at some interval other than that of collecting the virt info from the virt backends. Is this unnecessary? If we remove the individual timeouts we would end up checking the results of an async call and deciding what to do as a result only when we collect new data. Which is the preferable behavior?